ktxspec.adoc

```diff
@@ -977,6 +977,9 @@ _dfDescriptorBlock_.
     - If <<_vkformat,`vkFormat`>> is not one of the `+*_HLG{,_*}+`
       formats and an HLG variant of that format exists,
       `transferFunction` should not be `KHR_DF_TRANSFER_HLG`.
+* If <<_vkformat,`vkFormat`>> is one of the `+*_[SU]INT{,*}+` formats
```

Comment:
Should the pattern be `+*_[SU]INT{,_*}+`?

Reply:
Yes. Thanks.

Reply:
Should we also say that primaries for these formats must be `KHR_DF_PRIMARIES_UNSPECIFIED`?

Reply:
Yes.